projects/linkerd2-proxy/build.sh

```diff
@@ -14,27 +14,20 @@
 # limitations under the License.
 #
 ################################################################################
-TARGET_PATH="./fuzz/target/x86_64-unknown-linux-gnu/release"
-BASE="$SRC/linkerd2-proxy/linkerd"
-BUILD_FUZZER="cargo +nightly fuzz build --features fuzzing"
 
-cd ${BASE}/addr/
-${BUILD_FUZZER}
-cp ${TARGET_PATH}/fuzz_target_1 $OUT/fuzz_addr
+BASE=$PWD
+FUZZERS=$(find . -name "fuzz")
```

Comment:
do you want only directories?

Reply:
Yeah. All of the fuzzers are placed in "fuzz" directories across the repo and we need to find those in order to build the fuzzers.

Reply:
I had to add some more conditions for some tagrets, so decided to abandon generalising it in favour of readability. However, it should be ready to go now!